apps/journeys-admin/src/components/Editor/Toolbar/JourneyDetails/JourneyDetailsDialog/JourneyDetailsDialog.tsx

```diff
@@ -193,12 +197,15 @@ export function JourneyDetailsDialog({
                       />
                     )}
                     popper={{
-                      modifiers: [
-                        {
-                          name: 'flip',
-                          enabled: false
+                      placement: smUp ? 'bottom' : 'top',
```

Comment:
NIT: can we change this so it's easier to follow,

```suggestion
                      placement: !smUp ? 'top' : 'bottom',
```